.circleci/config.yml

```diff
@@ -22,15 +22,17 @@ workflows:
           mapping: |
             .circle/.* lint_only false
             cmake/.* lint_only false
-            configs/.* lint_only false
+            configs/.* lint_only true
```

Comment:
May remove the `lint_only true` lines, since the default value of `lint_only` is true in `test.yml`

Reply:
I found that the setting seemed not working, #1621 for example.

Reply:
Is it because `config.yml` has been changed?

Reply:
No matter I add `docs/.* lint_only true` into `config.yml` or not, circle CI run `test_linux_onnxruntime`. I only changed docs in that PR.

Reply:
https://github.com/open-mmlab/mmocr/blob/27b6a68586b9a040678fe083bcf60662ae1b9261/.circleci/test.yml#L126
I found an example in mmocr:
```
pr_stage_lint:
    when: << pipeline.parameters.lint_only >>
    jobs:
      - lint:
          name: lint
          filters:
            branches:
              ignore:
                - dev-1.x
                - 1.x
```
I think we should also try the `filters` option

Reply:
> https://github.com/open-mmlab/mmocr/blob/27b6a68586b9a040678fe083bcf60662ae1b9261/.circleci/test.yml#L126 I found an example in mmocr:
> 
> ```
> pr_stage_lint:
>     when: << pipeline.parameters.lint_only >>
>     jobs:
>       - lint:
>           name: lint
>           filters:
>             branches:
>               ignore:
>                 - dev-1.x
>                 - 1.x
> ```
> 
> I think we should also try the `filters` option

It's a bit confusing. If `pipeline.parameters.lint_only` is `True`, then we should do lint check and do not care about the branch.

Reply:
@lvhan028 @AllentDan maybe #1621 is because of not changing the `base-revision` in circleci config
https://github.com/open-mmlab/mmdeploy/pull/1621/commits/b0c5513f533ae7d6aac4668549e770d722a4a8cf

Reply:
It'll be OK if the conflicts are resolved.